src/fava/core/ingest.py

```diff
@@ -110,6 +111,8 @@ def load_file(self) -> None:
 
         self.mtime = os.stat(self.module_path).st_mtime_ns
         self.config = mod["CONFIG"]
+        if "HOOKS" in mod:
```

Comment:
Set the list to the default above this line, I think otherwise it would be incorrect after the first reload after removing `HOOKS` from an import config.